### DISCUSSION TITLE
print("...") eller print(f"...")

### DISCUSSION THREAD

**Discussion by kaareskokebok** [jdforlag/kaares_kokebok_vgs#2] [Q&A] [answered]
Jeg lurer på når man bruker `print(f"...")`, og når man bruker bare `print("...")`.
F.eks. her er to kodelinjer med print som gir samme resultat.

```python
pris = 300
print("prisen er", pris)
print(f"prisen er {pris}")
```

**Comment by kej83** [comment, marked as answer]
Bruk det som er enklest til situasjonen. Til eksempelet ditt er det enklest med `print("prisen er", pris). Dersom du ønsker mer kontroll overformateringen, og spesielt hvis du har flere variabler, er f-streng å foretrekke. Se f.eks. dette eksempelet:
```python
navn = "Ole"
alder = 30
print(f"{navn} er {alder} år gammel.")
```
er enklere enn 
```python
navn = "Ole"
alder = 30
print(navn, "er", alder, "år gammel.")
```
